src/docs/contributing/common_content.mdx

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
id probably put this under the platforms stuff (and I think it duplicates a bit of that) since almost all of this is focused on the platform portals

Reply:
I'll move it - I purposely tried to not to duplicate and instead point to the technical explanation.